service/flags.go

```diff
@@ -39,14 +37,15 @@ func (s *stringArrayValue) Set(val string) error {
 }
 
 func (s *stringArrayValue) String() string {
-	return "[" + strings.Join(s.values, ",") + "]"
+	return "[" + strings.Join(s.values, ", ") + "]"
 }
 
 func flags() *flag.FlagSet {
 	flagSet := new(flag.FlagSet)
 	featuregate.Flags(flagSet)
 
-	configFlag = flagSet.String("config", defaultConfig, "Path to the config file")
+	flagSet.Var(configFlag, "config", "Locations to the config file(s), note that only a"+
+		" single (first) location can be set per flag entry e.g. `-config=file:/path/to/first --config=file:path/to/second`.")
```

Comment:
```suggestion
		" single location can be set per flag entry e.g. `-config=file:/path/to/first --config=file:path/to/second`.")
```

I think the `(first)` bit was in relation to changing arrays in configs through `--set`.

Reply:
Yeah, copy paste ftw :)))